--- a/tools/BUILD.bazel
+++ b/tools/BUILD.bazel
@@ -26,3 +26,17 @@
     name = "defs",
     srcs = ["defs.bzl"],
 )
+
+# Headers used by the CPython JIT stencil generator.
+alias(
+    name = "cpython_jit_builtin_resource_dir",
+    actual = select({
+        "@llvm//platforms/config:linux_x86_64_prebuilt": "@llvm-toolchain-minimal-22.1.4-linux-amd64//:builtin_resource_dir",
+        "@llvm//platforms/config:linux_aarch64_prebuilt": "@llvm-toolchain-minimal-22.1.4-linux-arm64//:builtin_resource_dir",
+    }),
+)
+
+alias(
+    name = "cpython_jit_kernel_headers",
+    actual = "@kernel_headers//:kernel_headers_directory",
+)
